pkg/metrics/metrics.go

```diff
@@ -371,6 +373,45 @@ the maximum possible share value.`,
 	)
 )
 
+type LocalQueueMetricsConfig struct {
+	Enabled            bool
+	LocalQueueSelector labels.Selector
+}
+
+var lqMetricsConfigSingleton *LocalQueueMetricsConfig
+
+func SetLocalQueueMetrics(lqMetricsConfig *LocalQueueMetricsConfig) {
+	once.Do(func() {
+		lqMetricsConfigSingleton = lqMetricsConfig
+	})
+}
+
+var once = sync.Once{}
```

Comment:
Why do we need this?
Do you assume the situation where Configuration is updated during manager running?